### PR TITLE
Add Langarnia to showcase 

### PR DESCRIPTION
**Pull request by zoltanferenczfi**
## Checklist

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have read the [Showcase Requirements](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js)
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Showcase addition

* Title: *Langarnia*
* Description: *Langarnia is a 3D fantasy game designed to teach languages.*
* Website:   https://www.langarnia.com
* Source: 
* Tags:
  * `educational`
  * `game`

### CODE REVIEW

**Comment by zoltanferenczfi** [comment]
Hi All,
It was not clear for me how can I provide my logo file, so I added it here. Can you please place it to where it belongs?
Thanks!

langarnia

File: src/data/showcases.js [hunk 1]
```diff
@@ -1235,6 +1235,15 @@ export const Showcases = [
...
     tags: ["analytics"],
   },
+  {
+    title: "Langarnia",
+    description:
+      "Langarnia is a 3D fantasy game designed to teach languages.",
+    preview: require("./showcase/langarnia.png"),
+    website: "https://www.langarnia.com",
+    source: null,
+    tags: ["educational", "game"],
```
Comment (rphair) [inline src/data/showcases.js:1245]:
```suggestion
    tags: ["game"],
```
The `educational` tag is for education _about Cardano_.  As it says in the `showcases.js` file you are amending:
> Educational projects that will help you onboarding to Cardano.

**Review by rphair — COMMENTED** [review]
We can't upload new files via GitHub to your branch, so it would require a lot of manual work to duplicate your branch locally & commit that file for you.  You can upload it more easily here: https://github.com/zoltanferenczfi/developer-portal/tree/patch-1/src/data/showcase

I would not recommend uploading that square logo because, as you can see if you look on the Developer portal, the logo box width is over 2x the height, so you need something more like a wide rectangle (aspect ratio about 2.1:1).  The figure image might work if you have some styled text to the right of it, for instance.

**Comment by zoltanferenczfi** [comment]
Thanks for the guidance, I have uploaded / commited a new logo file.

**Review by rphair — APPROVED** [review]
Had to resolve last merge conflict manually... with a lot of water under the bridge... so I hope @rdlrt @os11k @fill-the-fill we can confirm or trash this soon 🤩

**Comment by rdlrt** [comment]
Agree with @rphair's [comment](https://github.com/cardano-foundation/developer-portal/pull/1005#pullrequestreview-1358971987) above

**Comment by rphair** [comment]
yes @rdlrt I still feel that way, but was just "going with the flow" from @katomm's last review since he was the one that suggested keeping the NFT categories trim in the first place.  With no news from @zoltanferenczfi about whether there is a novel Cardano token economy created by this game I would just as soon let it drop.

**Comment by zoltanferenczfi** [comment]
Dear All,

No news from me? Well, it is very hard to say anything about the insulting comments here about my project.

As a summary:
- Langarnia is a 3D RPG style fantasy game, developed in Unity, which is "a cross-platform game engine", but when I tagged my project as a game, this tag got removed, saying, it is not a game. Well, if it is not a game, I don't know what it is. You need a keyboard, a mouse or an xbox controller to control this software and walk around with your animated 3D charachter in a 3D fantasy world to talk to NPC-s in order to achieve your missions. In my opinion, it is a game, in yours, it is not.
- Langarnia is designed to teach foreign languages to anyone in a colorful and playful way. I tagged my project as educational. For me, teaching languages / anything is education, in yours, it is not. It would only deserve the educational tag, if it would teach blockchain and crypto, teaching languages is not education it seems. (ps: this game would also be able to teach kids about crypto and blockchain, we would just need to write up a proper study material with someone who is an expert in teaching crypto in an easy way)
- Langarnia was built to run on the Cardano blockchain, because we believed in the "going for #1" and will be the "future financial system of the world" and nevertheless "we are Cardano and we are here to change the world" and also "we are here to bank the unbanked". So everyone definitely need a Cardano address on the blockchain, a wallet set up in order to play this game. They need to learn how to set up their wallet, they need to learn how to make a transfer, they would also need to buy / get an NFT in order to play the game. We use blockfrosft.io to query the chain and NMKR.io to mint the NFT-s. This game cannot be played without someone being a user of the Cardano blockchain, but you are saying we don't meet the minimum requirement to get listed, as this product is not built on Cardano. 
- Now, the latest requirement would be to have a token and a use case for the token. Now, when everthing, even ADA is titled as a security and people getting sued left and right in the crypto space, we don't feel confident releasing and selling our tokens. Use case we would have in mind, but selling our tokens is not safe. 

As a summary, we thought, a team who spends their own free time and money to build such a product that I believe no other blockchain has and would be special, will be welcomed in the Cardano space with open arms, but we just faced the opposite. 
We thought it would be welcomed if we release the full product first, and only afterwards we release the utility token and not vica versa. We have never asked for money, never went to the Catalyst Fund, never asked anything from the foundation, never scammed or rugged anybody, wanted to create something great, but basically the message of the whole space was "go away".
I think the comment above where you guys write "hope to trash it soon" with a :D smiley face says it all.
In the last 1 year we reached out to all the big names, popular community members and influencers in the Cardano space in order to get some recognission, nobody ever answered to us, never managed to get a simple like on any of our posts, nobody followed us on Twitter or made a retweet. All these would have been free, just a press a button, but it seems rugs / scams / monkey pictures are value, teaching languages is not. 

So these are the news from me and now you can "trash" this item.

Regards,
Zoltan

**Comment by rphair** [comment]
Since I'm the one who used the word "trash" in a recent posting (as only one of two possibilities in a listing that went a long time without updates from the contributor, including some clarification I asked for directly), I'll accept responsibility for this reaction.

@zoltanferenczfi I approved this request as a sign of good faith and was especially hopeful that this project... or any other Cardano project... would eventually fulfill the enormous potential of using NFTs or other native tokens as part of an interactive game economy.

The fact that Langarnia doesn't do this yet doesn't mean the scenario you describe above... with a token merely required for participation... is without value.  I think I can speak for all reviewers in saying that we appreciate the sometimes unrewarding work you describe, regardless of the outcome of this submission.

I do think you are looking entirely on the negative side of a balanced response and taking what I said far out of context.  A response about the "token economy" question would have been much more welcome than focusing on the challenging words we've used in the course of a mainly positive review.

**Comment by zoltanferenczfi** [comment]
Hi Robert!

Many thanks for your kind support!

Zoltan

**Comment by katomm** [comment]
The pull request has not received the required approvals and is closed. However, it can be opened again. Details on: https://developers.cardano.org/docs/portal-contribute/#how-are-pull-requests-reviewed

In any case, the discussion can be continued here.

**Pull request closed without merging**